README.md

```diff
@@ -27,6 +27,7 @@ Some of the available features include:
 - Alarms and notifications
 - Account and device management
 - Email and SMS support
+- TOTP multi-factor authentication (Tested with Google Authenticator)
```

Comment:
Please revert this. We can't really list all available features in this list. It's only major ones.

Reply:
I removed the line